wordpress-mobile/WordPress-iOS
# UL&S: Update the password "eye" icon tint color in the old UI

Fixes #14538 
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/343

This PR is a color adjustment to the eye icon. 

### Screenshots
####  iPhone 5s, iOS 12
| Before | After |
| --- | --- |
| <a href="https://user-images.githubusercontent.com/1062444/88292918-e3269d80-ccbf-11ea-8a40-af041f8b12d0.png"><img src="https://user-images.githubusercontent.com/1062444/88292918-e3269d80-ccbf-11ea-8a40-af041f8b12d0.png" width="350" /></a> | <a href="https://user-images.githubusercontent.com/1062444/88854009-4334ac80-d1b6-11ea-8865-ed936f388172.png"><img src="https://user-images.githubusercontent.com/1062444/88854009-4334ac80-d1b6-11ea-8865-ed936f388172.png" width="350" /></a> |

#### iPhone 11 Pro, iOS 13. The bugfix in both modes.
| Light mode | Dark mode |
| --- | --- |
| <a href="https://user-images.githubusercontent.com/1062444/88855596-bf2ff400-d1b8-11ea-8c4b-2a1108ce28c9.png"><img src="https://user-images.githubusercontent.com/1062444/88855596-bf2ff400-d1b8-11ea-8c4b-2a1108ce28c9.png" width="350" /></a> | <a href="https://user-images.githubusercontent.com/1062444/88855635-ca831f80-d1b8-11ea-9bf2-08b29df7d6c9.png"><img src="https://user-images.githubusercontent.com/1062444/88855635-ca831f80-d1b8-11ea-9bf2-08b29df7d6c9.png" width="350" /></a> |

### To test:
It is recommended to test this PR on an iOS 12 device if you have one available. (I do not).

Dev testing notes: to save yourself some time with testing and building, you can optionally point the podfile to use a local build, because there is some hacking around to test this PR if you don't have an iOS 12 device available. Check out the Auth PR branch, point the podfile to `pod 'WordPressAuthenticator', :path => '../WordPressAuthenticator-iOS'`.

#### Test case 1, option A: build on an iOS 12 device
1. `rake dependencies`
2. Build and run.
3. Log In > Continue with WordPress.com. Observe the OnePassword icon color is updated on the old screen. Go back.
4. Log In > Site Address > enter a valid URL > Continue.
5. Username / password screen will show icons on the right side with the same tint color.

#### Test case 1, option B: hack it and test on an iOS 12 simulator
1. Check out the Auth PR wordpress-mobile/WordPressAuthenticator-iOS#332
2. Modify the WPiOS podfile to run Auth locally
3. [Comment out the OnePasswordEnabled check ](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/develop/WordPressAuthenticator/Extensions/WPStyleGuide%2BLogin.swift#L42-L44)
4. Log In > Site Address > enter a valid URL > Continue.
5. Username / password screen will show icons on the right side with the same tint color.

#### Test case 2
Run this on an iOS 13 device or sim. Your choice.

1. Reset your environment
2. `rake dependencies`
3. Build and run
4. Log In > Site Address > enter a valid URL > Continue.
5. The "eye" icon should match the tint color found on the OnePassword "lock" icon above it.

PR submission checklist:
